rb/lib/selenium/webdriver/common/logger.rb

```diff
@@ -83,32 +91,59 @@ def io
       #
       # @param [Array, Symbol] id
       #
-      def ignore(id)
-        Array(id).each { |ignore| @ignored << ignore }
+      def reject(id)
+        Array(id).each { |key| @reject << key }
+      end
+      alias ignore reject
```

Comment:
What do you think about using splat arguments for those?

```suggestion
      def reject(*ids)
        @reject += ids
      end
      alias ignore reject
```

Reply:
Hmm, would be nice to support `ignore(:foo, :bar, :baz)` as well as `ignore(%i[foo bar baz])`

Reply:
but I guess if choosing one, arguments are better than Array?

Reply:
I think it's the perfect case for splat arguments.

Reply:
```
        @ignored += Array(ids).flatten
```